packages/client-discord/src/voice.ts

```diff
@@ -397,11 +397,14 @@ export class VoiceManager extends EventEmitter {
                         const wavBuffer =
                             await this.convertOpusToWav(inputBuffer);
 
-                        console.log("starting transcription");
-                        const text = await this.runtime
-                            .getService(ServiceType.TRANSCRIPTION)
-                            .getInstance<ITranscriptionService>()
-                            .transcribe(wavBuffer);
+                        const transcriptionService =
+                            this.runtime.getService<ITranscriptionService>(
+                                ServiceType.TRANSCRIPTION
+                            );
+
+                        const text =
+                            await transcriptionService.transcribe(wavBuffer);
```

Comment:
if (!transcriptionService) {
                                    throw new Error(
                                        "Transcription generation service not found"
                                    );
                                }